src/main/java/de/ellpeck/rockbottom/api/item/ItemTool.java

```diff
@@ -72,4 +70,19 @@ public void onTileBroken(IWorld world, int x, int y, TileLayer layer, AbstractEn
             this.takeDamage(instance, player, 1);
         }
     }
+
+    @Override
+    public boolean onInteractWith(IWorld world, int x, int y, TileLayer layer, double mouseX, double mouseY, AbstractEntityPlayer player, ItemInstance instance) {
```

Comment:
this should be in ItemChisel, this is exactly what you said you didn't like about minecraft.

Reply:
This is not what I said I don't like about Minecraft actually.
I can put it into a separate class just for the Chisel if you want, but if that's the case then I'll remove the Tool Property for Chisel